en/docs/learn/admin-api.md

```diff
@@ -0,0 +1,273 @@
+Sign in to the API Publisher as a user whose role includes `Internal/publisher`. Follow the steps given below: 
+
+1. Sign in to the API Publisher portal at `https:// <WSO2_OB_APIM_HOST>:9443/publisher` with `creator/publisher` privileges.
+   You can use the credentials for `mark@gold.com`.
```

Comment:
This page should go to the 'try out' section, there we cannot mention Mark. Mark is created for testing purposes only.